certificate-authority/service/grpc/signCertificate.go

```diff
@@ -22,25 +26,111 @@ func (s *CertificateAuthorityServer) validateRequest(csr []byte) error {
 	return nil
 }
 
+func (s *CertificateAuthorityServer) updateSigningIdentityCertificateRecord(ctx context.Context, updateSigningRecord *pb.SigningRecord) error {
+	var found bool
+	now := time.Now().UnixNano()
+	err := s.store.LoadSigningRecords(ctx, updateSigningRecord.GetOwner(), &store.SigningRecordsQuery{
+		CommonNameFilter: []string{updateSigningRecord.GetCommonName()},
+	}, func(ctx context.Context, iter store.SigningRecordIter) (err error) {
+		for {
+			var signingRecord pb.SigningRecord
+			ok := iter.Next(ctx, &signingRecord)
+			if !ok {
+				break
+			}
+			fmt.Printf("updateSigningRecord: %v, %v\n", updateSigningRecord.GetPublicKey(), signingRecord.GetPublicKey())
```

Comment:
this log should probably go to some debug logger and not to stdout

Reply:
no it's debug log :)